### PR TITLE
Problems 1

### PR DESCRIPTION
**Pull request by UladBohdan**
Слоўнік і нататкі па перакладзе - тут:
https://docs.google.com/document/d/1gW8Z4fcFTKQFpv_hC44niSqE9vobKvVrvqJvXAvdceY/edit#
(непублічны доступ)

### CODE REVIEW

File: problems_1/problems_1.tex [hunk 1]
```diff
@@ -0,0 +1,264 @@
+\documentclass[12pt,twoside]{article}
+\setlength{\textwidth}{166mm}   % ширина текста
```
Comment (KlicOgogo) [inline problems_1/problems_1.tex]: Please, create a file for preamble

File: problems_1/problems_1.tex [hunk 2]
```diff
@@ -0,0 +1,264 @@
...
+
+\begin{itemize}
+
+\item[1.]
+
+Вылучыце ўмову і вынік тэарэмы, сфармулюйце яе з дапамогай звязкі <<Калі $\ldots$, то $\ldots$>>: \\
+\ruText{ Выделив условие и заключение теоремы, сформулируйте её посредством связки <<Если $\ldots$, то $\ldots$>>: }
+\begin{itemize}
+
+\item[(а)] Для таго, каб функцыя была дыферэнцавальнай ў некаторай кропцы, неабходна, каб яна была непарыўнай у гэтай кропцы; \\
```
Comment (KlicOgogo) [inline problems_1/problems_1.tex]: I think, redefining enumerate is better, but it's not necessary

File: problems_1/problems_1.tex [hunk 3]
```diff
@@ -0,0 +1,264 @@
...
+\end{itemize}
+
+
+
+
+\end{document}
+
+
+
+$\boldsymbol{\mathfrak{A}}, \boldsymbol{\mathfrak{B}}, \boldsymbol{\mathfrak{C}}, \boldsymbol{\mathfrak{D}},
```
Comment (KlicOgogo) [inline problems_1/problems_1.tex]: Why this code is placed after \end{document}?

File: problems_1/problems_1.tex (hunk 1 above)
Comment (UladBohdan) [reply problems_1/problems_1.tex]: Done.

File: problems_1/problems_1.tex (hunk 2 above)
Comment (UladBohdan) [reply problems_1/problems_1.tex]: You mean to avoid hardcoding Cyrillic letters?

File: problems_1/problems_1.tex (hunk 3 above)
Comment (UladBohdan) [reply problems_1/problems_1.tex]: I don't know! Removed it - nothing has changed 😂

Comment (KlicOgogo) [reply problems_1/problems_1.tex]: 😂😂😂

File: problems_1/problems_1.tex (hunk 2 above)
Comment (KlicOgogo) [reply problems_1/problems_1.tex]: Cyrillic letters and numbers for \item

File: problems_1/problems_1.tex (hunk 1 above)
Comment (KlicOgogo) [reply problems_1/problems_1.tex]: Didn't you forget to push it?

Comment (UladBohdan) [reply problems_1/problems_1.tex]: I replied before pushing, yes

File: problems_1/problems_1.tex (hunk 2 above)
Comment (KlicOgogo) [reply problems_1/problems_1.tex]: https://tex.stackexchange.com/questions/139462/controlling-enumerate-nesting-level

Comment (UladBohdan) [reply problems_1/problems_1.tex]: ok I'll try to do that

Comment (UladBohdan) [reply problems_1/problems_1.tex]: I did not see you link. I've done it in a slightly different way.

**Review by KlicOgogo — APPROVED** [review]

**Pull request merged**